src/components/Panels/PropertiesPanel/ModuleProperty/ProjectProperty/ProjectProperty.vue

```diff
@@ -80,6 +81,20 @@ import { getProjectName } from '#/simulator/src/data/save'
 import { toggleLayoutMode } from '#/simulator/src/layoutMode'
 import simulationArea from '#/simulator/src/simulationArea'
 import InputGroups from '#/components/Panels/Shared/InputGroups.vue'
+import { ref } from '@vue/reactivity'
+import { onMounted } from '@vue/runtime-core'
+const circuitId = ref(0)
+const circuitName = ref('Untitled-Cirucit')
+
+onMounted(() => {
+    // checking if circuit or tab is switched
+    setInterval(() => {
```

Comment:
we'd need a better way to do this, setInterval won't be a good option here,

Reply:
will create an issue for it to be looked at afterward.